src/app/components/PrizeField/PrizeField.module.css

```diff
@@ -0,0 +1,41 @@
+.Primary {
+  display: grid;
+  grid-auto-flow: column;
+  justify-content: left;
+}
+
+.Primary__prize {
+  border: none;
+  background-color: var(--first-prize-field-color);
+  display: grid;
+  align-content: center;
+  place-items: center;
+  color: var(--text-color);
+  width: 7.8em;
+  height: 5.4em;
+}
+
+.Primary__bar {
+  width: 1.8em;
+  background-color: var(--first-prize-spacebar-color);
+}
```

Comment:
These variant styles do seem to have overlapping properties which can be moved to a "base" css class